### PR TITLE
Small fixes to get migrations to run from clean database

### PR DESCRIPTION
**Pull request by dwhenry**

### CODE REVIEW

File: db/migrate/20140114144500_signin_permissions_must_be_lowercase.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class SigninPermissionsMustBeLowercase < ActiveRecord::Migration
   def up
-    SupportedPermission.update_all({ name: 'signin' }, { name: 'Signin' })
+    SupportedPermission.where({ name: 'Signin' }).update_all({ name: 'signin' })
```
Comment (benlovell) [inline db/migrate/20140114144500_signin_permissions_must_be_lowercase.rb]: I know you're doing a like-for-like here, but could we drop the surrounding braces for the option-style arguments? They're not necessary.

File: db/migrate/20150212133251_fix_user_update_permission.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class FixUserUpdatePermission < ActiveRecord::Migration
   def up
-    SupportedPermission.update_all({ grantable_from_ui: false }, { name: 'user_update_permission' })
+    SupportedPermission.where({ name: 'user_update_permission' }).update_all({ grantable_from_ui: false })
```
Comment (benlovell) [inline db/migrate/20150212133251_fix_user_update_permission.rb]: Same here with regards the surrounding hash braces.

**Comment by dwhenry** [comment]
@benlovell braces removed

**Comment by benlovell** [comment]
@dwhenry Can you squash please and I'll get this merged. Thanks!

**Comment by dwhenry** [comment]
@ben done, will merge before I create the PR in the future.

**Comment by benlovell** [comment]
Thanks!

**Pull request merged**